.github/workflows/coverage.yml

```diff
@@ -38,6 +38,11 @@ jobs:
         uses: ocaml/setup-ocaml@v2
         with:
           ocaml-compiler: ${{ matrix.ocaml-compiler }}
+          opam-depext-flags: --with-test # doesn't work
```

Comment:
If it doesn't work why still pass it then?

Reply:
It's an opam bug that eventually should be fixed: https://github.com/ocaml/opam/issues/5836. The additional `apt install` step in our CI should really be redundant since there's also a `depext` for it.